packages/wrangler/src/inspect.ts

```diff
@@ -124,8 +124,8 @@ export default function useInspector(props: InspectorProps) {
       server: serverRef.current,
       clientTracking: true,
     });
-    wsServerRef.current.on("connection", (ws: WebSocket) => {
-      if (wsServerRef.current.clients.size > 1) {
+    wsServerRef.current.on("connection", function (ws: WebSocket) {
+      if (this.clients.size > 1) {
```

Comment:
`this` is set to `wsServerRef.current` but not if we use a fat arrow...

Reply:
that's strange.